src/plugins/data/server/ui_settings.ts

```diff
@@ -703,6 +703,27 @@ export function getUiSettings(): Record<string, UiSettingsParams<unknown>> {
       }),
       schema: schema.boolean(),
     },
+    [UI_SETTINGS.AUTOCOMPLETE_VALUE_SUGGESTION_METHOD]: {
+      name: i18n.translate('data.advancedSettings.autocompleteValueSuggestionMethod', {
+        defaultMessage: 'Autocomplete value suggestion method',
+      }),
+      type: 'select',
+      value: 'terms_enum',
+      description: i18n.translate('data.advancedSettings.autocompleteValueSuggestionMethodText', {
+        defaultMessage:
+          'The method used for querying suggestions for values in KQL autocomplete. Select terms_enum to use the Elasticsearch terms enum API for improved autocomplete suggestion performance. Select terms_agg to use an Elasticsearch terms aggregation. {learnMoreLink}',
+        values: {
+          learnMoreLink:
+            '<a href="https://www.elastic.co/guide/en/kibana/current/kibana-concepts-analysts.html#autocomplete-suggestions" target="_blank" rel="noopener">' +
```

Comment:
Do we have to hardcode the full URL?

Reply:
I think we have to here, it looks like this is how the other links work in this file.